### PR TITLE
Remove aphrodite

### PR DESCRIPTION
**Pull request by tamorim**

### CODE REVIEW

File: render/index.js [hunk 1]
```diff
@@ -12,12 +11,9 @@ if (ExecutionEnvironment.canUseDOM) {
   const language = state.context.getIn(['culture', 'language'])
   addLocaleData(window.ReactIntlLocaleData[language])
   console.log('Welcome to Render')
-  StyleSheet.rehydrate(window.renderedClassNames)
-  ReactDOM.render(Root(store, locale), document.getElementById('render-container'))
```
Comment (firstdoit) [inline render/index.js]: Wait, what? Wheres the client-side rendering?

Comment (tamorim) [inline render/index.js]: Oops! I'mma fix client side rendering

**Comment by tamorim** [comment]
@firstdoit Fixed

**Comment by firstdoit** [comment]
Magic! 🎉  LGTM

**Pull request merged**